pkg/controller/node/controller_utils.go

```diff
@@ -263,12 +263,12 @@ func nodeExistsInCloudProvider(cloud cloudprovider.Interface, nodeName types.Nod
 	return true, nil
 }
 
-func recordNodeEvent(recorder record.EventRecorder, nodeName, nodeUID, eventtype, reason, event string) {
+func recordNodeEvent(recorder record.EventRecorder, nodeName, namespace, nodeUID, eventtype, reason, event string) {
 	ref := &clientv1.ObjectReference{
 		Kind:      "Node",
 		Name:      nodeName,
 		UID:       types.UID(nodeUID),
-		Namespace: "",
+		Namespace: namespace,
```

Comment:
even if node is non-namespace resource,it should belong to default namespace or it's namespace value is empty string.do no harm.If we add namespace to node in the future,it will works.